rsts/community/roadmap.rst

```diff
@@ -6,35 +6,36 @@ Roadmap
 
 How the community works?
 =========================
-Flyte is used actively in production at multiple companies. We pride ourselves at being extremely customer focused and cares deeply about a high quality customer experience. Thus we always
-prioritize stability, reliability, observability and maintainability over raw feature development. Features are developed usually in response to specific use cases and user scenarios. That being said,
-we are proactively thinking about the evolution of the system and how we want to keep adapting to changing requirements. Thus most of our changes reflect future development scenarios and in
-cases where we feel rapid prototyping would enable us to discover potential pitfalls or uncover hidden use cases we would proactively develop features, behind feature flags.
+Flyte is used actively in production at multiple companies. We pride ourselves on being extremely customer-focused and care deeply about a high quality customer experience. Thus, we always prioritize stability, reliability, observability, and maintainability over raw feature development. 
 
-Thus, it is extremely important to let the community know about your use cases and how we could adapt parts of Flyte to meet those requirements. We welcome collaboration and contributions, but please follow our Contribution Guidelines. 
+The features are developed usually in response to specific use cases and user scenarios. That being said, we are proactively thinking about the evolution of the system and how we want to keep adapting to the changing requirements. Thus, most of our changes reflect future development scenarios, and in cases where we feel rapid prototyping would enable us to discover potential pitfalls or uncover hidden use cases, we would proactively develop features, behind feature flags.
+
+Thus, it is extremely important to let the community know about your use cases and how we could adapt parts of Flyte to meet those requirements. We welcome collaboration and contributions, but please follow our Contribution Guidelines.
```

Comment:
```suggestion
Thus, it is extremely important to let the community know about your use cases and how we could adapt parts of Flyte to meet those requirements. We welcome collaborations and :ref:`contributions <...>`.
```
Can we add a sphinx directive to https://docs.flyte.org/en/latest/community/contribute.html, and reference it here?